knative-operator/deploy/resources/quickstart/serverless-application-quickstart.yaml

```diff
@@ -83,7 +83,7 @@ spec:
       review:
         failedTaskHelp: >-
           This task isn’t verified yet. Try the task again, or [read
-          more](https://docs.openshift.com/container-platform/4.11/applications/odc-viewing-application-composition-using-topology-view.html#odc-scaling-application-pods-and-checking-builds-and-routes_viewing-application-composition-using-topology-view)
+          more](https://docs.openshift.com/container-platform/4.12/applications/odc-viewing-application-composition-using-topology-view.html#odc-scaling-application-pods-and-checking-builds-and-routes_viewing-application-composition-using-topology-view)
```

Comment:
We could do it in a separate but it is fine.

Reply:
@pierDipi we have 4.11 showing in console js files we want to touch that too?

Reply:
Where are the js files?

I'm doing the index dockerfile update now

Reply:
Done the index dockerfile: https://github.com/openshift-knative/serverless-operator/pull/1795/commits/596393c54383fc7c34d383cec70b8975a6628b55 and https://github.com/openshift-knative/serverless-operator/pull/1795/commits/3bbf4d69a17c44a851287c108b0f87ddaa1e8191

Reply:
I found the js file

Reply:
we should take that hardcoded value as env variable injected based on project.yaml

Reply:
But I'm not familiar with how these JS tests work/run, so I'll defer that update to later on

Reply:
I have an attempt to fix/improve JS ui tests, but I will push them in a different PR.

Reply:
Yeah better in a new one let's not expand here.